scale_build/clean.py

```diff
@@ -25,6 +25,10 @@ def clean_packages():
             os.makedirs(path)
 
 
+def clean_diversions():
+    shutil.rmtree(DIVERT_DIR, ignore_errors=True)
```

Comment:
You are nuking builder's `/var/trash` here, i assume you want this for the scale image itself ? In that case we need to fix it's path below

Reply:
Thanks again